projects/ng-core-tester/src/app/app.component.ts

```diff
@@ -19,6 +19,8 @@ import { TranslateService } from '@ngx-translate/core';
 import { CoreConfigService, RecordEvent, RecordService, TitleMetaService } from '@rero/ng-core';
 import { BsLocaleService } from 'ngx-bootstrap/datepicker';
 import { ToastrService } from 'ngx-toastr';
+import { AppMenuService } from './service/app-menu.service';
+import { MenuItem } from 'projects/rero/ng-core/src/lib/menu/menu-item';
```

Comment:
Should be imported from `@rero/ng-core`.

Reply:
done